### DISCUSSION TITLE
Shall I use tsfresh for EEG data

### DISCUSSION THREAD

**Discussion by blrk** [blue-yonder/tsfresh#855] [Q&A] [answered]
Hi, I am working in EEG data. I am extracting the features using python code. Does tsfresh supports feature extraction in EEG?

**Comment by nils-braun** [comment, marked as answer]
Hi! I think there are two dimensions to your question:
* Does tsfresh technically work with EEG data? Yes. As long as you have the time series data in a format applicable to tsfresh, it will work.
* Are the extracted features meaningful for EEG data? This depends more on your target, than on your data. In principle I would propose to try it out :-) (maybe on a smaller subset of your data is too large). The feature extraction methods in tsfresh come from various domains - I am very positive that some of them will also be useful for your use case.

Question to the community: does someone have a nice use-case with EEG data to show? 

PS: it might also be a good idea to have a look into papers citing tsfresh. We have over 150 papers that used tsfresh, maybe there is one that has a similar use case?

**Reply by blrk** [reply]
Awesome, Ok, I will try it out. It's very nice to know that 150 papers used tsfresh.

**Reply by nils-braun** [reply]
Thanks for trying it out! 
I did find this: https://gist.github.com/F-A/ebd0cf72fb4e8b43d6d278db776f5824

And this: https://www.researchgate.net/publication/347827761_Detection_of_an_Autism_EEG_Signature_From_Only_Two_EEG_Channels_Through_Features_Extraction_and_Advanced_Machine_Learning_Analysis/references

Maybe it helps!

**Reply by blrk** [reply]
Thank you so much
